### PR TITLE
Added preprocessing.mdx italian translation

### PR DESCRIPTION
**Pull request by nickprock**
## What does this PR do?

Italian translation of doc related to the preprocessing of :hugs: Transformers.
* updated _toctree.yml
* added preprocessing.mdx

## Before submitting
- [X] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [X] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [X] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [X] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).

See issue: [#17459](https://github.com/huggingface/transformers/issues/17459)

@omarespejel 
@sgugger

### CODE REVIEW

**Comment by mfumanelli** [comment]
Hi @nickprock! Thank you for this translation 🚀🚀
Can I ask you if you can edit these things?
- Un modello non comprende il testo piano --> un modello non comprende un testo grezzo
-  CLS and SEP (classifier and separator) - --> CLS e SEP (classificatore e separatore)
- just before the "Audio" chapter there's a problem if you see the doc preview, where it says: ```py >>> batch_sentences = [ ... "But what about second breakfast?", ... "Don't think he knows about second breakfast, Pip.", ... "What about elevensies?", ... ] >>> encoded_input = tokenizer(batch_sentences, padding=True, truncation=True, return_tensors="tf") >>> print(encoded_input) {'input_ids': , 'token_type_ids': , 'attention_mask': } ``` I see the same problem in the English doc, you can take a cue from the Spanish doc where the problem has been solved
- E' importante che la frequenza --> È importante che la frequenza
- il modello Wav2Vec2 msu questo dataset --> il modello Wav2Vec2 su questo dataset
- Usa il metodo in 🤗 Datasets' cast_column per alzare della frequenza di campionamento a 16kHz --> Usa il metodo di 🤗 Datasets cast_column per alzare la frequenza di campionamento a 16kHz
- uno 0 - interpretato come silenzaio --> uno 0 - interpretato come silenzio
- Pad and truncate --> Pad e truncate
- il primo campione ga una sequenza --> il primo campione ha una sequenza
- LCrea una funzione che preprocesserà il dataset --> Crea una funzione che preprocesserà il dataset
- per applicare al volo la trasformzazione --> per applicare al volo la trasformazione
- Il Tokenizer per processarfe i testi. --> Il Tokenizer per processare i testi.
- e tokenizza il in to labels --> e tokenizza il testo in labels
- Fantastico, ora dovreste -> Fantastico, ora dovresti

Thank you 🌈🌈

**Comment by nickprock** [comment]
@mfumanelli thanks for the review 😊

**Comment by sgugger** [comment]
cc @omarespejel

**Review by sgugger — APPROVED** [review]
Sorry, I was misinformed and believed @omarespejel needed to review these PRs, but he does not. Apologies for the long wait and thanks a lot for your contributions!
